website/versioned_docs/version-6.x.x/schema-generator/schema-generator-getting-started.mdx

```diff
@@ -142,3 +142,9 @@ val customDef = TopLevelObject(query, Query::class)
 
 toSchema(config, listOf(customDef))
 ```
+
```

Comment:
we are already highlighting this in 
https://opensource.expediagroup.com/graphql-kotlin/docs/schema-generator/writing-schemas/arguments/ 
at the end of the page -- maybe we can add complementary details

Reply:
Working on it.

Reply:
Added only related issue.